chaos_genius/views/anomaly_data_view.py

```diff
@@ -426,7 +426,7 @@ def _get_dimensions_values(
 
     # series_type strings are in format {dimension1 == value1, dimension2 == value2,}
     # create a default dict mapping each dimension to a list of their values
-    dimension_values_dict = defaultdict(list)
+    dimension_values_dict: DefaultDict[str, List[str]] = defaultdict(list)
```

Comment:
Using set instead of the list in the defaultdict can improve the performance and can make you remove few line as well.

Reply:
Done in 8adb09d365b841af928e4b280d61da5d4818161a

Reply:
Let me know once you will test as well. I will merge the PR.